src/redis/object.c

```diff
@@ -268,7 +268,7 @@ robj *createHashObject(void) {
     o->encoding = OBJ_ENCODING_LISTPACK;
     return o;
 }
-
+/*
```

Comment:
Remove instead of commenting out?
I don't see the value in having commented out code, especially with no explanation surrounding it. I personally think that a better way to document stuff is by filing an issue / writing some `.md` explanation somewhere. You can even link to those from here.